ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main__arm.yaml

```diff
@@ -0,0 +1,46 @@
+base_images:
+  os:
+    name: centos
+    namespace: openshift
+    tag: stream9
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: prow/Dockerfile
+  from: os
+  to: krkn-hub-arm
+releases:
+  initial:
+    integration:
+      name: "4.14"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.14"
+      namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: krkn-hub-tests
```

Comment:
think we need to rename tis to be different than other test in the main file

Reply:
Ack, done. Thanks.